autofill-parser/gradle.properties

```diff
@@ -1,6 +1,7 @@
 GROUP=com.github.androidpasswordstore
 VERSION_NAME=1.0.0
 POM_ARTIFACT_ID=autofill-parser
+POM_ARTIFACT_DESCRIPTION=Android library for low-level parsing of Autofill structures
```

Comment:
```suggestion
POM_ARTIFACT_DESCRIPTION=Android library for AutofillService implementations that handles the low-level view structure parsing
```

Reply:
Doesn't really feel like an improvement, I'll merge my version for now and we can update it when something more explanatory occurs to either of us.